recipes-bsp/optee-ftpm/optee-ftpm-iot2050_0.1.bb

```diff
@@ -27,25 +22,12 @@ SRC_URI[wolfssl.sha256sum] = "a68c301fa0ee6197158912d808c4258605a2d001e458fd9582
 
 S = "${WORKDIR}/ms-tpm-20-ref-${SRCREV}"
 
-DEPENDS += "optee-os-tadevkit-iot2050"
-
-OPTEE_FTPM_BUILD_ARGS = " \
-    TA_CPU=cortex-a53 \
-    TA_CROSS_COMPILE=${CROSS_COMPILE} \
-    CFG_FTPM_USE_WOLF=y \
-    TA_DEV_KIT_DIR=/usr/lib/optee-os/export-ta_arm64 \
-    CFG_TEE_TA_LOG_LEVEL=2 \
-    "
-
-TEMPLATE_FILES = "debian/rules.tmpl"
-TEMPLATE_VARS += "OPTEE_FTPM_BUILD_ARGS"
-
-do_prepare_build() {
-    rm -rf ${S}/debian
-    cp -r ${WORKDIR}/debian ${S}/
-
-    deb_add_changelog
+OPTEE_NAME = "${MACHINE}"
```

Comment:
This will cause troubles for QEMU, see https://groups.google.com/d/msgid/isar-users/333a258f-708b-cf6b-7e7d-196b2d643bf1%40siemens.com